### PR TITLE
Fix/case archive

### PR DESCRIPTION
**Pull request by jonatanhanson**
Displays a list with all cases in the archive and fixed on click navigation for each case. Also made various fixes such as displaying correct input values for the selected case.

### CODE REVIEW

File: source/store/CaseContext.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ export function CaseProvider({ children }) {
       const [latestCase] = cases.sort(
         (c1, c2) => c2.attributes.updatedAt - c1.attributes.updatedAt
       );
-      return latestCase.attributes;
+      return latestCase;
```
Comment (jwinding) [inline source/store/CaseContext.js:25]: Jag ändrade också detta, på ett aningen annat sätt. Får se hur det krockar, om det krockar

File: source/screens/TaskScreen.js [hunk 2]
```diff
@@ -31,96 +32,77 @@ const ListHeading = styled(Heading)`
...
 `;
 
-class TaskScreen extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      activeTasks: [],
-    };
-  }
+const TaskScreen = ({ navigation }) => {
```
Comment (jwinding) [inline source/screens/TaskScreen.js:35]: Ändra namn Task --> Case överallt för att vara mer konsekvent

File: source/screens/TaskScreen.js [hunk 3]
```diff
@@ -31,96 +32,77 @@ const ListHeading = styled(Heading)`
...
+        Array.isArray(response.data.data) && response.data.data
+          ? sortTasksByLastUpdated(response.data.data)
+          : []
+      );
     } catch (error) {
-      console.log('Tasks not found: ', error);
+      console.error(`Get cases Error: ${error}`);
     }
-  };
+  }, [user]);
```
Comment (TeddyAndersson) [inline source/screens/TaskScreen.js]: A suggestion for this function is to move the get request for cases to the caseContext so we can reuse the functionality. 

The activeCase state could in that case also be set in the context, if we want to use or update ActiveCases somewhere else in the app.

File: source/store/CaseContext.js [hunk 4]
```diff
@@ -90,7 +89,7 @@ export function CaseProvider({ children }) {
    * Function for sending a put request towards the case api endpoint, updating the currently active case
    * @param {obj} data a object consiting of case user inputs.
    */
-  const updateCurrentCase = (data, status, currentStep) => {
+  const updateCurrentCase = async (data, status, currentStep) => {
```
Comment (TeddyAndersson) [inline source/store/CaseContext.js:91]: Might be a good idea to add a try catch in this function in case the put request fails.

**Pull request merged**